rued-esp32/src/services.rs

```diff
@@ -293,21 +295,21 @@ pub fn display(
         mem::forget(backlight); // TODO: For now
     }
 
-    let baudrate = 26.MHz().into();
+    let mut spi_config = SpiConfig::new();
+    spi_config.duplex = Duplex::Full;
+    let _ = spi_config.baudrate(24.MHz().into());
     // let baudrate = 40.MHz().into(); // Not supported on ESP32
 
     let spi = SpiDeviceDriver::new_single(
```

Comment:
How can I share this instance with the same instance in the main thread (used for SD card access?)